tableau-databricks/connection-builder.js

```diff
@@ -18,10 +18,14 @@ limitations under the License.
 (function dsbuilder(attr) {
 	var params = {};
 
+  var vendorDefined = {	
+		attributeHttpPath: "v-http-path"	
+	}
```

Comment:
Do we need Tableau help for the backwards compat?
Can we just write here
```
if (connectionHelper.attributeDatabase is set AND vendorDefined.attributeHttpPath is not set) {
  // for old workbooks
  params["HTTPPATH"] = attr[connectionHelper.attributeDatabase];
} else if (connectionHelper.attributeDatabase is not set AND vendorDefined.attributeHttpPath is set) {
  // for new workbooks
  params["HTTPPATH"] = attr[vendorDefined.attributeHttpPath];
} else if (connectionHelper.attributeDatabase is set AND vendorDefined.attributeHttpPath is set) {
  // for new workbooks with multiple catalogs
  params["<CATALOG>"] = connectionHelper.attributeDatabase;
  params["HTTPPATH"] = attr[vendorDefined.attributeHttpPath];
}

Reply:
Actually, scrap the last `else if`. You told me that when dbname is not specified, it just uses GetCatalogs and we're good.

Reply:
IIRC the problem occurred before the JS invocation, but I'm not sure if I'm remembering that correctly.